.github/workflows/test.yml

```diff
@@ -0,0 +1,50 @@
+name: Test
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  linux:
+    name: "Node ${{ matrix.node }} on Linux: Test and Lint"
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node:
+          - 10
+          - 12
```

Comment:
Maybe we should add 14 and 16 directly? Given that 10 is already out of support (but we probably should only drop it after Karma core drops it) and 12 goes out of support in [May](https://nodejs.org/en/about/releases/).

Reply:
Acknowledged. Let's address this more abstractly in: https://github.com/karma-runner/karma-chrome-launcher/issues/242.